gxutil/pm.go

```diff
@@ -695,12 +695,12 @@ func calledWithPathSeparator() bool {
 		return false
 	}
 
+	if strings.HasSuffix(trimmedArg, string(os.PathSeparator)+"gx") {
```

Comment:
It might be better to use https://golang.org/pkg/path/filepath/#Split in here instead.